common-pages/src/EventSearchTypeFilter.tsx

```diff
@@ -0,0 +1,182 @@
+﻿//******************************************************************************************************
+//  EventSearchTypeFilter.tsx - Gbtc
+//
+//  Copyright © 2023, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  02/02/2023 - C. Lackner
+//       Generated original version of source code.
+//  06/18/2024 - A. Karrar
+//       Configured Filter to be usable in GemStone
+//
+//******************************************************************************************************
+import React from 'react';
+import 'moment';
+import _ from 'lodash';
+import { OpenXDA } from '@gpa-gemstone/application-typings';
+
+
+interface IProps {
+    setFilter: (types: number[]) => void;
+    Height?: number;
+    eventTypes: OpenXDA.Types.EventType[];
+    evtTypeStatus: 'loading' | 'idle' | 'error' | 'changed' | 'unintiated';
+    eventTypeFilter: number[];
+}
```

Comment:
(1) Consistent capitalization in Props
(2) Renamed Props for Clarity
(3) Removed LoadingState
```suggestion
    SetSelectedTypeIDs: (typeIDs: number[]) => void;
    Height?: number;
    EventTypes: OpenXDA.Types.EventType[];
    SelectedTypeID: number[];
```